docs/01_overview/02_core_concepts.md

```diff
@@ -10,7 +10,7 @@ An account is a human-readable name that is stored on the blockchain. It can be
 
 ### Wallets
 
-Wallets are clients that store keys that may or may not be associated with the permissions of one or more accounts. Ideally, a wallet has a locked (encrypted) and unlocked (decrypted) state that is protected by a high entropy password. The Antelope/leap repository comes bundled with a CLI client called `cleos` that interfaces with a lite-client called `keosd` and together, they demonstrate this pattern.
+Wallets are clients that store keys that may or may not be associated with the permissions of one or more accounts. Ideally, a wallet has a locked (encrypted) and unlocked (decrypted) state that is protected by a high entropy password. The Antelope/leap repository, which EOS blockchain is built on, comes bundled with a CLI client called `cleos` that interfaces with a lite-client called `keosd` and together, they demonstrate this pattern.
```

Comment:
suggestion: add IO to Antelope and add link
Antelope/leap --> \[AntelopeIO/leap](https://github.com/AntelopeIO/leap)